templates/graylog/0/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+graylog-datavolume:
+    net: none
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    volumes:
+        - /graylog/data
+        - /graylog/logs
+        - /graylog/plugins
+    entrypoint: /bin/true
+    image: busybox
+mongodb-datavolume:
+    net: none
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    volumes:
+        - /data/db
+    entrypoint: /bin/true
+    image: busybox
+mongodb:
+    image: "mongo:3"
+    volumes_from:
+        - mongodb-datavolume
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.sidekicks: mongodb-datavolume
+elasticsearch:
+    image: "elasticsearch:2"
+    command: "elasticsearch -Des.cluster.name='graylog'"
+graylog:
+    restart: always
+    environment:
+        GRAYLOG_PASSWORD_SECRET: ${graylog_secret}
+        GRAYLOG_ROOT_PASSWORD_SHA2: ${graylog_password}
+        GRAYLOG_REST_TRANSPORT_URI: ${graylog_rest_transport_uri}
+        GRAYLOG_REST_LISTEN_URI: ${graylog_rest_listen_uri}
+        GRAYLOG_WEB_LISTEN_URI: ${graylog_web_listen_uri}
+        GRAYLOG_WEB_ENDPOINT_URI: ${graylog_web_endpoint_uri}
+        GRAYLOG_WEB_ENABLE_GZIP: ${graylog_web_enable_gzip}
+    image: "graylog2/server:2.0.3-2"
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.sidekicks: graylog-datavolume
+    volumes_from:
+        - graylog-datavolume
+    links:
+        - mongodb:mongo
+        - elasticsearch:elasticsearch
+lb:
```

Comment:
Depending on purpose ;) So we get rid lb, and finish ports forwarding manually ? seems like a good plan.